website/forms.py

```diff
@@ -33,3 +34,16 @@ def __init__(self, *args, **kwargs):
 
 
 
+# Create Add Record Form
+class AddRecordForm(forms.ModelForm):
+	first_name = forms.CharField(required=True, widget=forms.widgets.TextInput(attrs={"placeholder":"First Name", "class":"form-control"}), label="")
+	last_name = forms.CharField(required=True, widget=forms.widgets.TextInput(attrs={"placeholder":"Last Name", "class":"form-control"}), label="")
```

Comment:
No need to use `required=True` here as ModelForm use Model Field validations already